source/iostreams.tex

```diff
@@ -7566,32 +7689,149 @@
 
 \rSec3[stringbuf.members]{Member functions}
 
+\pnum
+The member functions getting the underlying character sequence
+all refer to a \tcode{high_mark} value,
+where \tcode{high_mark} represents the position
+one past the highest initialized character in the buffer.
+Characters can be initialized by writing to the stream,
+by constructing the \tcode{basic_stringbuf}
+passing a \tcode{basic_string} argument, or
+by calling one of the \tcode{str()} member functions
```

Comment:
Regarding str(): We don't want the parens unless we're actually talking about an expression. Here, we're just referring to a member function called "str", so no parens, please.

Reply:
I've been trying to make this point in LWG but some people think the parents help them know it's a function :confounded:

Should I add it to the specification style guide?

Reply:
We already have it:
https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#describing-function-calls

Reply:
Totally: `str()` is wrong - it implies the `str` member function that takes no parameters, and there can only be one of those in this context :).  `str` is clarified by "member functions" in this case, so there should be no confusion.

Fixed.